CompilerSource/JDI/src/System/macros.cpp

```diff
@@ -153,7 +153,7 @@ inline void jdip::macro_function::preparse(string val, error_handler *herr)
         value.push_back(mv_chunk(val.c_str(), push_from, i-push_from)); // Push it onto our value
       while (is_useless_macros(val[++i]));
       push_from = i; // Store current position just in case something stupid happens
-      if (!is_letter(val[i])) { herr->error("Expected parameter name following '#' token; `" + val + "'[" + ::toString(i) + "] is not a valid identifier character"); continue; }
+      if (!is_letter(val[i])) { herr->error("Expected parameter name following '#' token; `" + val + "'[" + ::toString((int)i) + "] is not a valid identifier character"); continue; }
```

Comment:
I think this is the relevant change https://github.com/enigma-dev/enigma-dev/blob/GL3.3NormalMatrix/CompilerSource/JDI/src/General/parse_basics.cpp